Wikipedia/Code/TalkPageFindInPageSearchController.swift

```diff
@@ -1,21 +1,21 @@
 import Foundation
 
 /// Locates occurrences of search term in a `[TalkPageCellViewModel]` hierarchy
-final class TalkPageFindInPageController {
+final class TalkPageFindInPageSearchController {
 
     // MARK: - Nested Types
 
     struct SearchResult {
 
         enum Location {
             // Term was located within the topic's title for topic at associated index
-            case topicTitle(topicIndex: Int)
+            case topicTitle(topicIndex: Int, topicIdentifier: ObjectIdentifier)
```

Comment:
I added them because I had a hunch they'd be useful for scrolling to specific positions. They aren't really used much now and I'm happy to remove them if they don't end up being necessary, but I'd prefer to wait until this feature is closer to completion before doing so to avoid repeated work if they are useful.